build.install4j

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<install4j version="6.1.2" transformSequenceNumber="5">
+<install4j version="6.1.5" transformSequenceNumber="5">
```

Comment:
@RoiEXLab I'll do a detailed review later, but FYI...  The Travis build currently uses install4j 6.1.2:

```
[WARNING] The version 6.1.2 of this install4j installation is lower than the version 6.1.5 of the IDE that saved the config file.
```

It's funny that you submitted this now because I am preparing some PRs to upgrade install4j to 6.1.5.  However, it will require multiple PRs across multiple repos (the install4j installer is located in the `triplea-game/assets` repo), so it may not happen quickly.

You may want to side-by-side install install4j 6.1.2 on your system and point your `install4jHomeDir` project variable there to verify your changes work as expected when building using that version.  If this PR gets merged before the install4j 6.1.5 upgrade PRs, I'll make sure to update this file as needed.